### PR TITLE
Paragon version upgraded to v18

### PR DESCRIPTION
**Pull request by Mashal-m**
**edx/paragon** version successfully upgraded to v18

### CODE REVIEW

File: src/users/CopyShowHyperLinks.test.jsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ describe('Copy Show Hyperlinks', () => {
     const copy = wrapper.find('a').at(0);
     const show = wrapper.find('a').at(1);
 
-    expect(copy.text()).toEqual('Copy ');
-    expect(show.text()).toEqual('Show ');
-    expect(wrapper.text()).toEqual('Copy Show ');
+    expect(copy.text()).toEqual('Copy');
+    expect(show.text()).toEqual('Show');
+    expect(wrapper.text()).toEqual('CopyShow');
   });
```
Comment (DawoudSheraz) [inline src/users/CopyShowHyperLinks.test.jsx:23]: Why is this changed?

File: src/users/SingleSignOnRecordCard.test.jsx [hunk 2]
```diff
@@ -108,7 +108,7 @@ describe.each(ssoRecordsData)('Single Sign On Record Card', (ssoRecordData) => {
         expect(text).toEqual(expires);
       } else {
         expect(text).toEqual(
-          value.length > 14 ? 'Copy Show ' : value,
+          value.length > 14 ? 'CopyShow' : value,
```
Comment (DawoudSheraz) [inline src/users/SingleSignOnRecordCard.test.jsx:111]: same here.

File: src/users/CopyShowHyperLinks.test.jsx (hunk 1 above)
Comment (Mashal-m) [reply src/users/CopyShowHyperLinks.test.jsx:23]: Due to a minor change of the paragon version in 16, tests we were failing. Fixed that

**Review by DawoudSheraz — APPROVED** [review]
Please squash before merge.
